future/transaction_test.go

```diff
@@ -889,19 +888,13 @@ func TestLogicSig(t *testing.T) {
 	key, err := mnemonic.ToPrivateKey(mn)
 	var pk crypto.MultisigAccount
 	require.NoError(t, err)
-	lsig, err := crypto.MakeLogicSig(program, args, key, pk)
+	lsig, err := crypto.MakeLogicSigAccountDelegatedMsig(program, args, pk, key)
 	require.NoError(t, err)
 
-	_, stxBytes, err := crypto.SignLogicsigTransaction(lsig, tx)
+	_, stxBytes, err := crypto.SignLogicsigTransaction(lsig.Lsig, tx)
```

Comment:
nit: `SignLogicSigAccountTransaction` would be preferred here

Reply:
Good find! fixed + renamed existing function to capitalize `Sig`: https://github.com/algorand/go-algorand-sdk/pull/450/commits/6c824ec8945946b83110656d8fe3c09f1ceff983